style.css

```
@@ -125,3 +125,227 @@ img {
     gap: 20px;
   }
 }
+
+.about {
+  background: var(--background-black, #171525);
```

```suggestion
  background: var(--background-black);
```

第2引数は代替値ですが、今回の場合ファイル上部で定義されていることが確実なので書かなくてよいかと思います。
CSS変数をJavaScriptで動的に書き換えたりする場合に役に立つものという認識です。

---

他の部分も同様です。